contributing.md

```diff
@@ -9,54 +11,121 @@ This document ***adds*** to this process, detailing what we have found useful
 repo and is where the ***key changes to the workflow*** will take place.
 
-### Step 1: _Confirm_ the Need in the New Story/Issue
+#### :arrow_right: Step 1: _Confirm_ the Need in the New Story/Issue
 The Product Owner (PO) will confirm the _need_, but if the issue is confirmed,
 the PO will also:
 
-#### A) Confirm that there is a set of acceptance criteria listed in the issue
+##### A) Confirm that there is a set of acceptance criteria listed in the issue
 + Acceptance Criteria should be in the form of a _checklist_ of items in the
   original post. This way it will appear in the issue list, giving an idea of
   how much of the acceptance criteria has been met when the issue is worked
-  on  
+  on
   ![checklist-in-issue-list-screenshot](https://user-images.githubusercontent.com/4185328/69977981-a422bc80-1523-11ea-9df2-8e892dd6f884.png)
 
 
 + If no acceptance criteria is present, the PO should either:
-  + Add the `needs-criteria` label to the issue to signal that this still
+  + :label: Add the `needs-criteria` label to the issue to signal that this still
     needs to be added before work can proceed; OR
   + Add acceptance criteria to the original post if they are intimately familiar
-    with what is required  
+    with what is required
     ![needs-criteria-label-screenshot](https://user-images.githubusercontent.com/4185328/69977426-b2240d80-1522-11ea-95f3-8de79ce516fb.png)
 
-#### B) Confirm that there is a proposed User Interface *(UI)* for the issue
+##### B) Confirm that there is a proposed User Interface *(UI)* for the issue
 
-Similar to `needs-criteria`, a `needs-ui` label should be applied to the issue
+:label: Similar to `needs-criteria`, a `needs-ui` label should be applied to the issue
 if no UI has been proposed and until a UI is agreed.
 
 _Note:_ This does not have to be the _final, unchangeable_ UI. Every UI is a
 starting point for testing with people using the app.
 
-### Step 2: Assign a Priority to the Issue
-:warning: A priority label should only be used for the issues expected to come up
+#### :arrow_right: Step 2: Assign a Priority to the Issue
+:warning: :label: A priority label should only be used for the issues expected to come up
 in the next 2 sprints or so.
 
-All other issues - should a rough priority for them be known at this stage - will
-be put into the `To Do` column of the
-[Github Project](https://github.com/dwyl/app/projects) in prioritised order.
-
 _We are testing whether this is able to prevent the notion that `priority-4` and
 `priority-5` labels are ["where issues go to die"](https://github.com/dwyl/app/issues/239#issuecomment-556105866)_.
 
+:bar_chart: Once an issue **has acceptance criteria, UI (where applicable) and
+a priority label**, it should be be added to the `To Do` column of the
+[Github Project](https://github.com/dwyl/app/projects) where ***issues are kept
+in prioritised order***.
+
+> The `To Do` column of the kanban board should only contain issues that are
```

Comment:
There is an outstanding task with me here to go through the current To Do column and ensure this is enforced